enterprise/backend/test/metabase_enterprise/serialization/v2/e2e/yaml_test.clj

```diff
@@ -158,6 +158,16 @@
                                                {:table_id      [:t    100]
                                                 :collection_id [:coll 100]
                                                 :creator_id    [:u    10]}))
+               :simple-model            (mapv #(update-in % [1 :refs] table->db)
```

Comment:
with the new `pre-update` in card, all implicit actions will be deleted during the generate part( I'm not sure why it calls update here but it is what it is).

The reason implicit actions got deleted is that the spec for `:card` in this test is a complex query with aggregation and breakout.

So I need to create a new spec that use a simple model so that our actions are safe from updates.

Reply:
Can you add a comment for this in the code? It would be almost impossible to know this without diving into this PR otherwise.

Reply:
yep, added here: https://github.com/metabase/metabase/pull/29021/commits/1d8c6582972870b47b21603782d8618639ef371d